_docs/setup/target_rate.md

```diff
@@ -193,29 +193,29 @@ If we set no from/to times, then our 24 hour period being looked at ranges from
+| `2023-01-01T06:30` | `2023-01-01T06:30` - `2023-01-01T07:00`, `2023-01-01T07:00` - `2023-01-01T07:30` | `true`                             | Both of our cheapest rates in the target range are in the past, so we must look for the next lowest combined rate. |![image](https://github.com/eugeneniemand/HomeAssistant-OctopusEnergy/assets/6813309/84057ecd-a612-4de0-b2a4-51e67ef7e146) <!-- Scenario 16 --> |
+| `2023-01-01T18:30` | None | `true`                            | There is no longer enough time available within our restricted time, so we have to wait until our target period has passed to look at the next evaluation period. ||
 
 If we set our from/to times to look over two days, from `20:00` to `06:00`, we then limit the period that we look at to overnight. The following table shows what this would be like.
 
-| current date/time  | period                                | `Re-evaluate multiple times a day` | reasoning |
-| ------------------ | ------------------------------------- | ---------------------------------- | --------- |
-| `2023-01-01T20:00` | `2023-01-01T23:30` - `2023-01-02T00:30`, `2023-01-02T05:00` - `2023-01-02T05:30` | `false`                            | Our lowest rate of 5 now falls between our overnight time period so is available. |
-| `2023-01-02T02:00` | `2023-01-01T23:30` - `2023-01-02T00:30`, `2023-01-02T05:00` - `2023-01-02T05:30` | `false`                            | Our lowest period is in the past, but we still have a rate in the future so our sensor will only come on once. |
-| `2023-01-02T02:00` | `2023-01-02T02:00` - `2023-01-02T02:30`, `2023-01-02T05:00` - `2023-01-02T05:30` | `true`                             | The rate of 5 is in the past, so we must look for the next lowest combined rate, which includes our half hour rate at 7. |
-| `2023-01-02T05:30` | None | `true`                             | There is no longer enough time available within our restricted time, so we have to wait until our target period has passed to look at the next evaluation period. |
+| current date/time  | period                                | `Re-evaluate multiple times a day` | reasoning | example |
+| ------------------ | ------------------------------------- | ---------------------------------- | --------- |---------|
+| `2023-01-01T20:00` | `2023-01-01T23:30` - `2023-01-02T00:30`, `2023-01-02T05:00` - `2023-01-02T05:30` | `false`                            | Our lowest rate of 5 now falls between our overnight time period so is available. |![image](https://github.com/eugeneniemand/HomeAssistant-OctopusEnergy/assets/6813309/077122c7-d12c-4d5c-8d87-d43e9e80dc16) <!-- Scenario 17 --> |
```

Comment:
This currently indicates a total of 1.5 hours not 1 or am I misunderstanding something